hal/targets/hal/TARGET_ARM_SSG/TARGET_BEETLE/TOOLCHAIN_IAR/.gitattributes

```diff
@@ -0,0 +1,4 @@
+*.o binary
+*.lib binary
+*.a binary
+*.ar binary
```

Comment:
I am adding binary libraries for the Cordio stack and platforms and I need git to know that are binaries. Any specific reason why I should remove it?